EIPS/eip-4758.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 4758
+title: Deactivate SELFDESTRUCT
+description: Deactivate SELFDESTRUCT by changing it to SENDALL, which does recover all funds to the caller but does not delete any contract code or storage.
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin), Dankrad Feist (@dankrad)
+discussions-to: https://github.com/ethereum/EIPs/issues/4759
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-03
+requires: 
+---
+
+# Deactivating SELFDESTRUCT
+
+## Summary
+
+This EIP renames the `SELFDESTRUCT` opcode to `SENDALL`, and replaces its functionality. The new functionality will be only to send all Ether in the account to the caller.
+
+## Motivation
+
+The `SELFDESTRUCT` opcode requires large changes to the state of an account, in particular removing all code and storage. This will not be possible in the future with Verkle trees: Each account will be stored in many different account keys, which will not be obviously connected to the root account (see [Verkle tree EIP](https://notes.ethereum.org/5HDhQXstTaKtVqVbS7S9yw)).
+
+This EIP implements this change. Applications that only use SELFDESTRUCT to retrieve funds will still work.
+
+## Specification
+
+ * The `SELFDESTRUCT` opcode is renamed to `SENDALL`, and now only immediately moves all ETH in the account to the target; it no longer destroys code or storage or alters the nonce
```

Comment:
Perhaps it also deserves explicit mention here that a `SENDALL` where the target is the invoker no longer burns the ETH in the account.